.github/workflows/clang-format-checker.yml

```diff
@@ -15,14 +15,16 @@ jobs:
       - name: Fetch LLVM sources
         uses: actions/checkout@v4
         with:
-          fetch-depth: 2
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          fetch-depth: 0 # Fetch all history
 
       - name: Get changed files
         id: changed-files
-        uses: step-security/changed-files@3dbe17c78367e7d60f00d78ae6781a35be47b4a1 # v45.0.1
+        uses: tj-actions/changed-files@v41
```

Comment:
This change was deliberate in https://github.com/microsoft/DirectXShaderCompiler/pull/7217 because the tj-actions version of the action had a significant security vulnerability.

I'm not sure what the correct fix is, but we should not revert back to the tj-actions version.

Reply:
Definitely was not intending to check this in. Initially didn't think everyone would see the PR if it was marked as a draft.